src/ekka_autocluster.erl

```diff
@@ -208,14 +208,14 @@ discover_and_join(Mod, Options) ->
             log_error("Register", ekka_cluster_strategy:register(Mod, Options)),
             case DeadNodes of
                 [] ->
-                    ?LOG(info, "no discovered nodes outside cluster", []),
+                    ?LOG(info, "all discovered nodes are alive", []),
                     case Res of
                         {error, _} -> error;
                         ok         -> ok;
                         ignore     -> ignore
                     end;
                 [_ | _] ->
-                    ?LOG(warning, "discovered nodes outside cluster: ~p", [DeadNodes]),
+                    ?LOG(warning, "discovered nodes are not responding: ~p", [DeadNodes]),
```

Comment:
I think it makes sense to lower severity of this message. It's a pretty normal situation when some discovered nodes are down, usually it doesn't require operator's intervention.

Reply:
Done.